apps/filebrowser/src/filebrowser/views.py

```diff
@@ -15,105 +15,118 @@
 from desktop.lib.conf import coerce_bool
-from desktop.lib.django_util import render, format_preserving_redirect
-from desktop.lib.django_util import JsonResponse
-from desktop.lib.export_csvxls import file_reader
+from desktop.lib.django_util import JsonResponse, format_preserving_redirect, render
 from desktop.lib.exceptions_renderable import PopupException
+from desktop.lib.export_csvxls import file_reader
 from desktop.lib.fs import splitpath
-from desktop.lib.fs.ozone.ofs import get_ofs_home_directory
 from desktop.lib.fs.gc.gs import get_gs_home_directory
+from desktop.lib.fs.ozone.ofs import get_ofs_home_directory
+from desktop.lib.fs.ozone.upload import OFSFineUploaderChunkedUpload
 from desktop.lib.i18n import smart_str
 from desktop.lib.paths import SAFE_CHARACTERS_URI, SAFE_CHARACTERS_URI_COMPONENTS
 from desktop.lib.tasks.compress_files.compress_utils import compress_files_in_hdfs
 from desktop.lib.tasks.extract_archive.extract_utils import extract_archive_in_hdfs
 from desktop.lib.view_util import is_ajax
 from desktop.views import serve_403_error
+from filebrowser.conf import (
+  ARCHIVE_UPLOAD_TEMPDIR,
+  ENABLE_EXTRACT_UPLOADED_ARCHIVE,
+  FILE_DOWNLOAD_CACHE_CONTROL,
+  MAX_SNAPPY_DECOMPRESSION_SIZE,
+  REDIRECT_DOWNLOAD,
+  SHOW_DOWNLOAD_BUTTON,
+  SHOW_UPLOAD_BUTTON,
+)
+from filebrowser.forms import (
+  ChmodFormSet,
+  ChownFormSet,
+  CopyFormSet,
+  EditorForm,
+  MkDirForm,
+  RenameForm,
+  RenameFormSet,
+  RestoreFormSet,
+  RmTreeFormSet,
+  SetReplicationFactorForm,
+  TouchForm,
+  TrashPurgeForm,
+  UploadArchiveForm,
+  UploadFileForm,
+)
+from filebrowser.lib import xxd
+from filebrowser.lib.archives import archive_factory
+from filebrowser.lib.rwx import filetype, rwx
+from hadoop.conf import UPLOAD_CHUNK_SIZE
 from hadoop.core_site import get_trash_interval
-from hadoop.fs.hadoopfs import Hdfs
 from hadoop.fs.exceptions import WebHdfsException
 from hadoop.fs.fsutils import do_overwrite_save
-from useradmin.models import User, Group
-
-from filebrowser.conf import ENABLE_EXTRACT_UPLOADED_ARCHIVE, MAX_SNAPPY_DECOMPRESSION_SIZE,\
-    SHOW_DOWNLOAD_BUTTON, SHOW_UPLOAD_BUTTON, REDIRECT_DOWNLOAD, FILE_DOWNLOAD_CACHE_CONTROL
-from filebrowser.lib.archives import archive_factory
-from filebrowser.lib.rwx import filetype, rwx
-from filebrowser.lib import xxd
-from filebrowser.forms import RenameForm, UploadFileForm, UploadArchiveForm, MkDirForm, EditorForm, TouchForm,\
-    RenameFormSet, RmTreeFormSet, ChmodFormSet, ChownFormSet, CopyFormSet, RestoreFormSet,\
-    TrashPurgeForm, SetReplicationFactorForm
-
+from hadoop.fs.hadoopfs import Hdfs
 from hadoop.fs.upload import HDFSFineUploaderChunkedUpload, LocalFineUploaderChunkedUpload
-from aws.s3.upload import S3FineUploaderChunkedUpload
-from azure.abfs.upload import ABFSFineUploaderChunkedUpload
-from desktop.lib.fs.ozone.upload import OFSFineUploaderChunkedUpload
+from useradmin.models import Group, User
+
+standard_library.install_aliases()
```

Comment:
Can remove this and future import line 45

Reply:
removed it